frontend/src/api.ts

```diff
@@ -30,6 +30,17 @@ export class ApiClient {
   constructor(baseUrl: string = "https://10ten.trap.show/api", authToken: string = "") {
```

Comment:
```suggestion
  constructor(baseUrl: string = "https://10ten.trap.show/api", authToken?: string) {
```
空文字列 `""` は JavaScript / TypeScript において falsy なので今のままの実装でも動くんですが、authToken が外部から渡されなかった場合に結局その変数を用いないのであれば空文字列による初期化を消してしまった方がコードの意図を掴みやすくなるかもしれません